src/main/groovy/com/blackducksoftware/integration/hub/detect/util/executable/ExecutableRunner.java

```diff
@@ -74,6 +75,24 @@ public ExecutableOutput execute(final Executable executable, final boolean runQu
         }
     }
 
+    public void executeToFile(final Executable executable, final File file) throws ExecutableRunnerException {
+        logger.debug(String.format("Running executable >%s", executable.getExecutableDescription()));
+        try {
+            final ProcessBuilder processBuilder = executable.createProcessBuilder();
+            final Process process = processBuilder.redirectOutput(file).start();
+            process.waitFor();
+            final ExecutableOutput errorOutput = new ExecutableOutput("", printStream(process.getErrorStream(), true, true));
```

Comment:
Having this all in one line may lead to some confusion if issues come up around the sub-processes